frontend/src/static/js/v_authorship.js

```diff
@@ -332,7 +373,17 @@ window.vAuthorship = {
 
   created() {
     this.initiate();
-    this.setInfoHash();
+  },
+
+  beforeDestroy() {
+    window.removeHash('authorshipFileTypes');
+    window.removeHash('authorshipFilesGlob');
```

Comment:
Can factor out all the removing of hashes to a separate method, and call the method in `beforeDestroy`.